srtcore/srt.h

```diff
@@ -358,6 +363,11 @@ struct CBytePerfMon
    uint64_t byteSndDrop;                // number of too-late-to-send dropped bytes
    uint64_t byteRcvDrop;                // number of too-late-to play missing bytes (estimate based on average packet size)
    uint64_t byteRcvUndecrypt;           // number of undecrypted bytes
+
+   uint32_t countBreak;                 // increased with every broken link in the group
+   uint32_t countActivate;              // increased with every activated link in the group
+   uint32_t countEager;                 // increased with every case when unstable link became stable again
+   uint32_t countSilence;               // increased with every case of switching a link back to idle
```

Comment:
Maybe better? 🤔 
```suggestion
   uint32_t countDeactivated;               // increased with every case of switching a link back to idle
```

Reply:
That looks ok, if you prefer. Just note I can't simply commit this suggestion because the name must be changed everywhere, including documentation.